player.js

```diff
@@ -0,0 +1,63 @@
+var Scrabble = require("./scrabble.js");
+var scrabble = new Scrabble();
+
+var Player = function(name) {
+  this.name =  name;
+  this.plays = [];
+};
+
+Player.prototype.play = function(word) {
+  if (this.hasWon()) {
+    return false
+  }
+  this.plays.push(word);
+}
+
+Player.prototype.wordScores = function() {
+  var scores = [];
+  for (word of this.plays) {
+    scores.push(scrabble.score(word));
+  }
+
+  return scores;
+}
+
+Player.prototype.totalScore = function() {
+  if (this.plays.length <= 0) {
+    return 0;
+  }
+
+  var current_scores = this.wordScores();
```

Comment:
Remember to use camelCase when naming variables, functions, etc.  I think you definitely get this but it's a hard habit to break after Ruby-ing for so long. 😉